src/dualize.jl

```diff
@@ -44,6 +44,11 @@ On each of these methods, the user can provide the following keyword arguments:
   * `ignore_objective`: a boolean indicating if the objective function should be
     added to the dual model. This is also useful for bi-level modelling, where
     the second level model is represented as a KKT in the upper level model.
+
+  * `assume_min_if_feasibility`: a boolean indicating if the objective function 
```

Comment:
This should also be a parameter of `dual_optimizer`

Reply:
Yep